Sming/project.mk

```diff
@@ -160,6 +163,7 @@ COMPONENT_VARS			:=
 COMPONENT_RELINK_VARS	:=
 COMPONENT_TARGETS		:=
 COMPONENT_DEPENDS		:=
+override undefine COMPONENT_PYTHON_REQUIREMENTS
```

Comment:
Doesn't work with MinGW. Why not just `COMPONENT_PYTHON_REQUIREMENTS :=` ...

Reply:
(Specifically, `undefine` not supported in GNU Make 3.81)

Reply:
The difference between an undefined vs. an empty `COMPONENT_PYTHON_REQUIREMENTS` is auto detection of 'requirements.txt', i.e. in the empty case no auto-detection is performed.

In the latest commit, I moved auto-detection before component evaluation. This way, the component can simply clear the variable if it wishes to suppress an auto-detected `requirements.txt`. No `undefine` necessary & successfully tested with make 3.8.